### ISSUE TITLE
Code of Conduct

### ISSUE THREAD

**Issue by annevk** [tabatkins/bikeshed#1588]
I'm not really sure what the precedent is for having a Code of Conduct for a single person project, but it still seems good to have one to me?

**Comment by tabatkins** [comment]
Yeah, seems reasonable. I'm happy to continue being benevolent-dictator and kicking out anyone who's a jerk, but might as well signal what sort of things are unacceptable.
